### PR TITLE
Levene uses t and approporiate df

### PR DESCRIPTION
**Pull request by JohnnyDoorn**
Fix https://github.com/jasp-stats/jasp-issues/issues/1641

### CODE REVIEW

File: R/ttestindependentsamples.R [hunk 1]
```diff
@@ -429,11 +429,11 @@ ttestIndependentMainTableRow <- function(variable, dataset, test, testStat, effS
 .ttestIndependentEqVarRow <- function(table, variable, groups, dataset) {
   levene <- car::leveneTest(dataset[[ .v(variable) ]], dataset[[ .v(groups) ]], "mean")
 
-  F  <- levene[1, "F value"]
-  df <- levene[1, "Df"]
+  t  <- sqrt(levene[1, "F value"])
```
Comment (vandenman) [inline R/ttestindependentsamples.R:432]: If you modify a function and encounter `.v()` (or `.unv()`), please remove it!

```r
levene <- car::leveneTest(dataset[[variable]], dataset[[groups]], "mean")
```

**Review by vandenman — CHANGES_REQUESTED** [review]
The changes look fine but can you please remove all `.v()` calls within `.ttestIndependentEqVarRow`?

File: R/ttestindependentsamples.R (hunk 1 above)
Comment (JohnnyDoorn) [reply R/ttestindependentsamples.R]: ah yes, done. I thought id just leave it how i found it, but now i know!

**Review by vandenman — APPROVED** [review]

**Comment by AlexanderLyNL** [comment]
I'm still not sure why it should be a t, furthermore, the resulting t will always be positive, which is not what a t-statistic should do.

**Comment by EJWagenmakers** [comment]
Isn't this because the Levene test for two groups effectively boils down to a t-test on the absolute deviations from the mean?

**Comment by AlexanderLyNL** [comment]
Yes, but by the same argument we should also change everytime an F(1, \nu) into a t. The problem is that this is not really true, because you miss a sign, when taking the square root, see https://github.com/jasp-stats/jaspTTests/pull/121/files#diff-9f0344e81555dc91009c55e1adc3dc2aacb84680a6437b720d46f375d85c5eb5L432. 

If we keep F, then we don't have to worry the sign.

File: R/ttestindependentsamples.R [hunk 2]
```diff
@@ -195,7 +195,7 @@ TTestIndependentSamples <- function(jaspResults, dataset = NULL, options, ...) {
   equalityVariance$showSpecifiedColumnsOnly <- TRUE
   equalityVariance$position <- 3
   equalityVariance$addColumnInfo(name = "variable", type = "string",  title = "")
-  equalityVariance$addColumnInfo(name = "F",        type = "number",  title = gettext("F"))
+  equalityVariance$addColumnInfo(name = "t",        type = "number",  title = gettext("t"))
```
Comment (AlexanderLyNL) [inline R/ttestindependentsamples.R:198]: Keep F

**Review by AlexanderLyNL — CHANGES_REQUESTED** [review]
Don't change the Levene test statistic into a t.

**Comment by EJWagenmakers** [comment]
I am missing something. If this is a test of equality of variance for *two* groups (?!), then the statistics *should* be a t -- not obtained by taking the square root of F, but just by doing a t-test on the absolute deviations. In our regular t-test we also report a t, not an F.

**Comment by AlexanderLyNL** [comment]
Yes the test is about equality of variances of two groups. In this setting it's about selecting between a Welch test or a Student's T test. The degrees of freedom of Levene's test should be 2-1, see

https://en.wikipedia.org/wiki/Levene%27s_test

**Comment by EJWagenmakers** [comment]
OK, but how is your remark a response to mine? I still don't get why it would be incorrect for a two-group comparison to report the t value from a comparison of the absolute deviations from the mean.

**Comment by AlexanderLyNL** [comment]
That wouldn't be a usual t statistic, because that's always positive. A standard F test makes sense, because the quantities are positive, which is what a Levene's test is.

**Comment by EJWagenmakers** [comment]
The quantities are positive, but it is the direction that determines the sign (i.e., X>Y or Y>X). This also means that you could do the test one-sided; for instance, if there is a control and a treatment condition, the test may be on whether the variance in the treatment condition is *larger* (not smaller) than that in the control condition. I realize the test is usually conducted in a more agnostic manner.

**Comment by vandenman** [comment]
Actually, I'm not sure if the request from the commenter in the original issue makes sense. They argue that because we do a t-test, we should report a t statistic and not an F statistic. But everywhere I google I find that it is customary to report the results of a Levene's test like so:

> Levene’s test indicated unequal variances (F = 3.56, p = .043), ...

or like so (from an SPSS manual):

> F(2, 77) = 4.58, p = .013

So to me, it seems more appropriate to show `F` and not `t`.

**Comment by AlexanderLyNL** [comment]
I don't have SPSS, but I found the following two examples:

- https://statistics.laerd.com/statistical-guides/independent-t-test-statistical-guide.php
- https://libguides.library.kent.edu/spss/independentttest Table in the output and "A"

File: R/ttestindependentsamples.R [hunk 3]
```diff
@@ -427,13 +427,13 @@ ttestIndependentMainTableRow <- function(variable, dataset, test, testStat, effS
 }
 
 .ttestIndependentEqVarRow <- function(table, variable, groups, dataset) {
-  levene <- car::leveneTest(dataset[[ .v(variable) ]], dataset[[ .v(groups) ]], "mean")
+  levene <- car::leveneTest(dataset[[ variable ]], dataset[[ groups ]], "mean")
 
-  F  <- levene[1, "F value"]
-  df <- levene[1, "Df"]
+  t  <- sqrt(levene[1, "F value"])
+  df <- levene[2, "Df"]
```
Comment (AlexanderLyNL) [inline R/ttestindependentsamples.R:433]: This should remain 1, see https://en.wikipedia.org/wiki/Levene%27s_test

**Comment by JohnnyDoorn** [comment]
Alright, so I move back to F, so that the only change in this PR is the different df that gets reported in the t-test assumption check?

**Comment by AlexanderLyNL** [comment]
The degrees of freedom for the Levene's test is 1, so as far as I know, nothing changes.

**Review by AlexanderLyNL — APPROVED** [review]
If unit test works, then this should be fine.

**Pull request merged**